src/govuk/components/footer/_index.scss

```diff
@@ -37,27 +31,20 @@
   }
 
   .govuk-footer__link {
+    @include govuk-link-common;
+
     @if ($govuk-use-legacy-palette) {
       &:link,
       &:visited {
-        color: $govuk-footer-link;
+        color: $govuk-footer-text;
       }
 
       &:hover,
       &:active {
-        color: $govuk-footer-link-hover;
+        color: #171819; // Only used with the legacy palette
```

Comment:
Not a big issue, but not sure how I feel about hardcoding the value rather than declaring it above. I know we're doing it because it's only ever used once now, with the legacy palette, but I wonder if someone looking at this file might see the values defined at the top and assume those are the only legacy values and miss this one?